extensions/amp-ad-network-oblivki-impl/OWNERS

```diff
@@ -0,0 +1,15 @@
+// For an explanation of the OWNERS rules and syntax, see:
+// https://github.com/ampproject/amp-github-apps/blob/master/owners/OWNERS.example
+
+{
+  rules: [
+    {
+      owners: [
+          {name: 'cathyxz', requestReviews: false},
```

Comment:
Do you really want to include *cathyxz* in owners? I felt she was here because that is what the example has.